src/controllers/v1/reports.js

```diff
@@ -0,0 +1,26 @@
+const reportService = require('@services/reports')
+
+module.exports = class Reports {
+	/**
+	 * Report filter list
+	 * @method
+	 * @name reportFilterList
+	 * @param {Object} req - request data.
+	 * @param {String} req.decodedToken.token - user token.
+	 * @returns {JSON} - filter list.
+	 */
+
+	async reportFilterList(req) {
```

Comment:
@sumanvpacewisdom controller itself is report, Just name the function as filterList

Reply:
@rakeshSgr its for label and value